build.gradle

```diff
@@ -1,159 +1,114 @@
-plugins {
-    id 'com.android.application' version '7.3.1'
-    id 'com.android.library' version '7.3.1' apply false
+ext {
+    assetsPath = '../../assets'
+    thirdPartyAssetsPath = '../../third_party/assets'
 }
 
-android {
-    namespace 'com.google.bigwheels'
-    compileSdk 32
+subprojects {
+    repositories {
+        mavenCentral()
+        google()
+    }
 
-    defaultConfig {
-        applicationId "com.google.bigwheels"
-        minSdk 29
-        targetSdk 32
-        versionCode 1
-        versionName "1.0"
+    apply plugin: "com.android.application"
 
-        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
-        externalNativeBuild {
-            cmake {
-                arguments '-DPPX_ANDROID=TRUE',
-                          '-DPPX_BUILD_TESTS=FALSE',
-                          '-DBUILD_TESTS=FALSE' // Required for OpenXR
-                cppFlags '-std=c++20'
-                if (project.hasProperty('DXC_PATH')) {
-                  arguments "-DDXC_PATH=$DXC_PATH"
-                }
-            }
-        }
-        multiDexEnabled true
+    dependencies {
+        implementation 'androidx.appcompat:appcompat:1.5.1'
+        implementation 'com.google.android.material:material:1.7.0'
+        testImplementation 'junit:junit:4.13.2'
+        androidTestImplementation 'androidx.test.ext:junit:1.1.4'
+        androidTestImplementation 'androidx.test.espresso:espresso-core:3.5.0'
     }
 
-    buildTypes {
-        release {
-            minifyEnabled false
+    buildDir = "../../build/android/${project.name}"
+
+    android {
+        namespace 'com.google.bigwheels'
+        compileSdkVersion 32
```

Comment:
nit: compileSdk? unless going back to the old naming is required)

Reply:
Changed, thanks! not sure why I put compileSdkVersion before